lab_3_generate_by_ngrams/main.py

```diff
@@ -76,6 +104,12 @@ def get_token(self, element_id: int) -> Optional[str]:
 
         In case of corrupt input arguments or arguments not included in storage, None is returned
         """
+        inv_storage = {identifier: element for element, identifier in self._storage.items()}
+
+        if not isinstance(element_id, int) or element_id not in inv_storage:
```

Comment:
It is better to place checks before the main logic

Reply:
fixed